reactor-core/src/main/java/reactor/core/publisher/FluxMap.java

```diff
@@ -104,7 +104,7 @@ public void onNext(T t) {
 
 			try {
 				v = Objects.requireNonNull(mapper.apply(t),
-						"The mapper returned a null value.");
+						"The mapper [" + mapper.getClass().getName() + "] returned a null value.");
```

Comment:
For better performance, we can concat the strings only after checking the return value is null. e.g.:
```java
Object value = mapper.apply(t);
if (value == null) {
    throw new NullPointerException("The mapper [" + mapper.getClass().getName() + "] returned a null value.");
}
```

Reply:
sure will change it , thanks @JamesChenX